src/components/SEO.tsx

```diff
@@ -25,6 +28,53 @@ const SEO = (): JSX.Element => {
         name="description"
       />
       <title>vaxxed.as | {translate("header.Scan your NZ COVIDpass")}</title>
+      <link
```

Comment:
No need to add the favicon, webmanifest, tile, and theme colours here. 

Please take a look at the `gatsby-config.js` file, there's where the magic happens.

Favicons are automatically generated as well. Just overwrite the SVG file at `./src/images/icon.svg`

Reply:
Ah yep sweet, never used gatsby before I usually just do meta tags myself